db/migrate/20190122160000_create_tags.rb

```diff
@@ -0,0 +1,10 @@
+# Create Tags Table
+
+class CreateTags < ActiveRecord::Migration[5.2]
+  def change
+    create_table :tags do |t|
+      ## 
+      t.string :title,null: false   
```

Comment:
Can we put a space after the comma here to improve readability?

Reply:
inserted space